services/management/ia/templates_service.go

```diff
@@ -151,13 +250,33 @@ func (svc *TemplatesService) loadFile(ctx context.Context, file string) ([]saas.
 	return rules, nil
 }
 
+func convertParamType(t alert.Type) iav1beta1.ParamType {
+	// TODO: add another types.
+	switch t {
+	case alert.Float:
+		return iav1beta1.ParamType_FLOAT
+	default:
+		return iav1beta1.ParamType_PARAM_TYPE_INVALID
+	}
+}
+
+func convertParamUnit(u string) iav1beta1.ParamUnit {
+	// TODO: check possible variants.
+	switch u {
+	case "%", "percentage":
+		return iav1beta1.ParamUnit_PERCENTAGE
+	default:
+		return iav1beta1.ParamUnit_PARAM_UNIT_INVALID
+	}
+}
```

Comment:
There is no any validation for unit on `saas.Rule` side, there it should be just non-empty string. Here we have `enum`.